hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/WriteFileThroughputBenchmark.java

```diff
@@ -0,0 +1,208 @@
+    implements Callable<Void> {
+
+  @Option(names = {"-o"},
+      description = "Ozone filesystem path",
+      defaultValue = "o3fs://bucket1.vol1")
+  private String rootPath;
+
+  @Option(names = {"-s", "--size"},
+      description = "Size of each generated files (in GB)",
+      defaultValue = "1")
+  private long fileSize;
+
+  @Option(names = {"-bl", "--block"},
+      description = "Specify the Block Size in MB",
+      defaultValue = "128")
+  private long blockSize;
+
+  @Option(names = {"-bu", "--buffer"},
+      description = "Size of buffer used store the generated " +
+          "key content",
+      defaultValue = "10240")
+  private int bufferSize;
+
+  @Option(names = {"-th", "--throttle"},
+      description = "Specify the Delay in Input/Output",
+      defaultValue = "0")
+  private int throttle;
+
+  @Option(names = {"-re", "--replication"},
+      description = "Specify the Replication factor",
+      defaultValue = "1")
+  private short replication;
+
+  @Option(names = {"--sync"},
+      description = "Optionally Issue hsync after every write " +
+          "Cannot be used with hflush",
+      defaultValue = "false"
+  )
+  private boolean hSync;
+
+  @Option(names = {"--flush"},
+      description = "Optionally Issue hsync after every write " +
+          "Cannot be used with hflush",
+      defaultValue = "false"
+  )
+  private boolean hFlush;
+
+  // For Generating the content of the files
+  private ContentGenerator contentGenerator;
+  // For Creating the required configurations for the file system
+  private OzoneConfiguration configuration;
+
+  private URI uri;
+
+  // variable to check if the user wants a delay
+  private boolean isThrottled;
+
+  private long expectedIoTimeNs;
+
+  private Timer timer;
+
+  private final ThreadLocal<FileSystem> threadLocalFileSystem =
+      ThreadLocal.withInitial(this::createFS);
+
+  public static final Logger LOG =
+      LoggerFactory.getLogger(WriteFileThroughputBenchmark.class);
+
+
+  public Void call() throws Exception {
+
+    init();
+    LOG.info("NumFiles=" + getTestNo());
+    LOG.info("Total FileSize=" + fileSize);
+    LOG.info("BlockSize=" + blockSize);
+    LOG.info("BufferSize=" + bufferSize);
+    LOG.info("Replication=" + replication);
+    LOG.info("Threads=" + getThreadNo());
+    LOG.info("URI Scheme Used=" + uri.getScheme());
+    if (hSync) {
+      LOG.info("Hsync after every write= True");
+    } else if (hFlush) {
+      LOG.info("Hflush after every write= True");
+    }
+
+    // Initialize the configuration variable with
+    // OzoneFS configuration
+    configuration = createOzoneConfiguration();
+
+    //Constructs a URI object by parsing the given string rootPath
+    uri = URI.create(rootPath);
+
+    // Disabling the file system cache
+    String disableCacheName = String.format("fs.%s.impl.disable.cache",
+        uri.getScheme());
+    print("Disabling FS cache: " + disableCacheName);
```

Comment:
use log.info()

Reply:
Thanks for the suggestion will make the change !!